### ISSUE TITLE
Question for the support of dhg for heterogeneous hypergraph 

### ISSUE THREAD

**Issue by mgao97** [iMoonLab/DeepHypergraph#9]
I am wondering whether dhg supports hypergraph datasets with heterogeneous graphs, which means that for each hyperedge, there are several types of entities.

**Comment by yifanfeng97** [comment]
Absolutely, a heterogeneous hyperedge will have a more powerful modeling ability. It is a good direction for DHG's future work. Can you provide more materials? Some related links like code repositories, papers, and datasets of existing heterogeneous structures?

**Comment by mgao97** [comment]
Thanks! I have no ability to offer materials of such characteristics, and looking forward to your findings!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "iMoonLab/DeepHypergraph"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月19日(星期一) 晚上7:08
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [iMoonLab/DeepHypergraph] Question for the support of dhg for heterogeneous hypergraph  (Issue #9)





 
Absolutely, a heterogeneous hyperedge will have a more powerful modeling ability. It is a good direction for DHG's future work. Can you provide more materials? Some related links like code repositories, papers, and datasets of existing heterogeneous structures?

**Comment by beating-1224** [comment]
There was a study on node classification task ,which proposed a graph neural network-based representation learning framework for heterogeneous hypergraphs. This work has been published in WSDM，named Heterogeneous Hypergraph Embedding for Graph Classification. It may give you some reference.
